dependencies.yaml

```diff
@@ -356,7 +356,7 @@ dependencies:
           - numpy>=1.23,<3.0a0
           - zarr
           # See https://github.com/zarr-developers/numcodecs/pull/475
```

Comment:
Is the PR linked above still relevant? I think it may be safe to remove this comment now.